solver/llbsolver/file/backend.go

```diff
@@ -179,14 +179,20 @@ func docopy(ctx context.Context, src, dest string, action pb.FileActionCopy, u *
 		return nil
 	}
 
-	ch, err := mapUserToChowner(u, idmap)
-	if err != nil {
-		return err
+	var ch copy.Chowner
+	if action.Owner != nil {
```

Comment:
this check here definitely isn't correct. First, action (including owner) is already parsed before calling this function and the result `*copy.User` passed to this function directly (with a nil check in map). Secondly, this object being nil doesn't carry any meaning as it is just a combination of two possibly nil pointers.

This issue looks to be regression from https://github.com/moby/buildkit/pull/1342